src/ResourceManager/Dns/Stack/Commands.Dns.Test/packages.config

```diff
@@ -0,0 +1,31 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<packages>
+  <package id="Hyak.Common" version="1.0.3" targetFramework="net45" />
+  <package id="Microsoft.Azure.Common" version="2.1.0" targetFramework="net45" />
+  <package id="Microsoft.Azure.Gallery" version="2.6.2-preview" targetFramework="net45" />
+  <package id="Microsoft.Azure.Common.Dependencies" version="1.0.0" targetFramework="net45" />
+  <package id="Microsoft.Azure.Management.Authorization" version="2.0.0" targetFramework="net45" />
+  <package id="Microsoft.Azure.Management.Dns" version="1.7.2-preview" targetFramework="net45" />
+  <package id="Microsoft.Azure.Management.ResourceManager" version="1.1.1-preview" targetFramework="net45" />
+  <package id="Microsoft.Azure.Management.Resources" version="2.20.0-preview" targetFramework="net45" />
+  <package id="Microsoft.Azure.Test.HttpRecorder" version="1.6.7-preview"  targetFramework="net45" />
+  <package id="Microsoft.Azure.Test.Framework" version="1.0.6179.26854-prerelease" targetFramework="net45" />
+  <package id="Microsoft.Bcl" version="1.1.9" targetFramework="net45" />
+  <package id="Microsoft.Bcl.Async" version="1.0.168" targetFramework="net45" />
+  <package id="Microsoft.Bcl.Build" version="1.0.14" targetFramework="net45" />
+  <package id="Microsoft.IdentityModel.Clients.ActiveDirectory" version="2.28.3" targetFramework="net45" />
+  <package id="Microsoft.Net.Http" version="2.2.28" targetFramework="net45" />
+  <package id="Microsoft.Rest.ClientRuntime" version="2.3.5" targetFramework="net45" />
```

Comment:
@markcowl can you please let me know what version should we use here..

Reply:
Since the .csproj already using common targets, these should not be needed in packages.config?

Reply:
thanks for pointing out. I will keep the test project as such. the module .csproj already uses common targets